test/integration.test.js

```diff
@@ -0,0 +1,36 @@
+const { expect } = require("chai").use(require("chai-as-promised"));
+const Erc721 = artifacts.require("TradeTrustERC721");
+const TitleEscrow = artifacts.require("TitleEscrow");
+
+contract("TradeTrustErc721", (accounts) => {
+  const shippingLine = accounts[0];
+  const beneficiary1 = accounts[1];
+
+  const merkleRoot = "0x624d0d7ae6f44d41d368d8280856dbaac6aa29fb3b35f45b80a7c1c90032eeb3";  
+  const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+  describe("TradeTrustERC721 Surrender Flow", () => {
+    let tokenRegistryInstanceWithShippingLineWallet;
+    let tokenRegistryAddress;
+
+    beforeEach("", async () => {
```

Comment:
Why got `""` de? I think can just skip that and put the async function in right?